src/main/java/kitchenpos/application/OrderService.java

```diff
@@ -1,110 +1,72 @@
-            throw new IllegalArgumentException("주문하려면 하나 이상의 메뉴가 필요합니다.");
-        }
-
-        final List<Long> menuIds = orderLineItems.stream()
-                .map(OrderLineItem::getMenuId)
-                .collect(Collectors.toList());
-
-        if (orderLineItems.size() != menuDao.countByIdIn(menuIds)) {
-            throw new IllegalArgumentException("등록되지 않은 메뉴는 주문할 수 없습니다.");
-        }
-
-        order.setId(null);
-
-        final OrderTable orderTable = orderTableDao.findById(order.getOrderTableId())
-                .orElseThrow(() -> new IllegalArgumentException("존재하지 않는 테이블은 주문할 수 없습니다."));
+    public CreateOrderResponse create(final CreateOrderRequest request) {
+        final OrderTable orderTable = getOrderTable(request);
+        final List<OrderLineItem> orderLineItems = getOrderLineItems(request);
 
-        if (orderTable.isEmpty()) {
-            throw new IllegalArgumentException("빈 테이블은 주문할 수 없습니다.");
-        }
+        final Order order = new Order(orderTable, orderLineItems);
+        final Order savedOrder = orderRepository.save(order);
 
-        order.setOrderTableId(orderTable.getId());
-        order.setOrderStatus(OrderStatus.COOKING.name());
-        order.setOrderedTime(LocalDateTime.now());
-
-        final Order savedOrder = orderDao.save(order);
-
-        final Long orderId = savedOrder.getId();
-        final List<OrderLineItem> savedOrderLineItems = new ArrayList<>();
-        for (final OrderLineItem orderLineItem : orderLineItems) {
-            orderLineItem.setOrderId(orderId);
-            savedOrderLineItems.add(orderLineItemDao.save(orderLineItem));
-        }
-        savedOrder.setOrderLineItems(savedOrderLineItems);
-
-        return savedOrder;
+        return CreateOrderResponse.from(savedOrder);
     }
 
-    public List<Order> list() {
-        final List<Order> orders = orderDao.findAll();
+    private OrderTable getOrderTable(CreateOrderRequest request) {
+        return orderTableRepository.findById(request.getOrderTableId())
+                                   .orElseThrow(() -> new IllegalArgumentException("존재하지 않는 테이블은 주문할 수 없습니다."));
+    }
 
-        for (final Order order : orders) {
-            order.setOrderLineItems(orderLineItemDao.findAllByOrderId(order.getId()));
-        }
+    private List<OrderLineItem> getOrderLineItems(CreateOrderRequest request) {
+        return request.getOrderLineItems()
+                      .stream()
+                      .map(item -> {
+                          final Menu menu = menuRepository.findById(item.getMenuId())
+                                                          .orElseThrow(() -> new IllegalArgumentException("등록되지 않은 메뉴는 주문할 수 없습니다."));
+                          return new OrderLineItem(menu, item.getQuantity());
+                      })
+                      .collect(Collectors.toList());
+    }
 
-        return orders;
+    public List<OrderResponse> list() {
+        return orderRepository.findAll().stream()
```

Comment:
```suggestion
        return orderRepository.findAll()
        .stream()
```
혹시 copy & paste의 결과인가요 list 메서드에서 동일하게 나오네요ㅋㅋㅋ